cookiecutter/main.py

```diff
@@ -36,11 +37,11 @@ def cookiecutter(
     :param extra_context: A dictionary of context that overrides default
         and user configuration.
     :param: overwrite_if_exists: Overwrite the contents of output directory
-        if it exists
+        if it exists.
```

Comment:
@Angus-Millar thanks very much for your contribution!
However, I think this is the only change I agree with... 😬